gateway-service-health/src/main/java/org/apache/knox/gateway/service/health/PingResource.java

```diff
@@ -72,16 +78,31 @@ private Response getPingResponse() {
     response.setHeader(CACHE_CONTROL, NO_CACHE);
     response.setContentType(CONTENT_TYPE);
     try (PrintWriter writer = response.getWriter()) {
-      writer.println(getPingContent());
+      writer.println(OK);
```

Comment:
Is this changing the behavior of the HEALTH service in an incompatible way? Or is it still always returning "OK"? Want to make sure we do not need to have a new API version (i.e., v2)?

Reply:
Ping always returns OK as before.